techniques/general/G60.html

```diff
@@ -2,112 +2,98 @@
+            <li><a href="#important-information">達成方法に関する重要な情報</a></li>
+            <li><a href="#applicability">適用 (対象) </a></li>
+            <li><a href="#description">解説</a></li>
+            <li><a href="#examples">事例</a></li>
+            <li><a href="#related">関連する達成方法</a></li>
+            <li><a href="#tests">検証</a></li>
          </ul>
       </nav>
-      <h1>Playing a sound that turns off automatically within three seconds</h1>
+      <h1>音声の再生を 3 秒以内に自動的に停止する</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://w3c.github.io/wcag/understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
-         </p>
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。</p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Applies to all technologies except those for voice interaction.</p>
-            <p>This technique relates to <span><a href="https://w3c.github.io/wcag/understanding/audio-control">Success Criterion 1.4.2: Audio Control</a> (Sufficient)</span>.
-            </p>
+            <h2>適用 (対象) </h2>
+            <p>音声のインタラクション機能があるものを除く全てのウェブコンテンツ技術</p>
+            <p>これは<span><a href="https://w3c.github.io/wcag/understanding/audio-control">達成基準 1.4.2: 音声の制御</a> (十分な達成方法) </span>に関連する達成方法である。</p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The purpose of this technique is to allow authors to play a sound on their
-               Web page but avoid the problem of users not being able to use their screen
-               readers due to interference by the content sound. It also allows the author
-               to avoid putting controls on the Web page to control the sound - and the problem
-               faced by users with screen readers in finding the control (when unable
-               to hear their screen reader).
+            <h2>解説</h2>
+            <p>
+               この達成方法の目的は、コンテンツ制作者がウェブページ上で音声を再生できるようにする一方で、利用者がその音声による干渉のためにスクリーンリーダーを使用できないという問題を回避することである。
+               また、コンテンツ製作者が音声を制御するコントロールをウェブページに配置することを回避し、スクリーンリーダーの利用者が、(スクリーンリーダーの音が聞こえない場合に) そのコントロールを見つける際に直面する問題を回避することもできる。
```

Comment:
具体的なレビューありがとうございましたmm

1文目の

> due to interference by the content sound.

こちらは、「content」をどう訳すか迷いましたが、「その音声による干渉のために」と訳してみました。

2文目については、「avoid」のかかり方（ to avoid putting controls ~ and the problem）がわかりづらいかなと考え、「配置することを回避し、〜を回避することもできる」としてみました。

また、

>  (when unable	to hear their screen reader)

の部分について、2.0の訳に少し近くなってしまいましたが修正してみました。

`スクリーンリーダーの利用者が、そのコントロールを見つける際に (スクリーンリーダーの音が聞こえない場合に) 直面する問題を回避することもできる。`

という訳と悩んだのでフィードバックいただければ幸いです。